package.json

```
@@ -0,0 +1,27 @@
+{
+  "name": "tcp-server",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "jest",
+    "test:watch": "jest --watch",
+    "lint": "eslint .",
+    "pretest": "npm run lint"
```

```suggestion
    "pretest": "npm run lint",
    "client": "node lib/client.js"
```
Scripts make it easier to test out files :)